main.go

```diff
@@ -278,8 +278,8 @@ func discoverNetwork(network string, queue chan func(context.Context), exporter
 				addr, _ := net.LookupAddr(ip) // #nosec
 				hostname := strings.TrimRight(getFirst(addr), ".")
 				if hostname == "" {
-					logrus.Error("missing reverse record for ", ip)
-					continue
+					logrus.Info("Missing reverse record for ", ip, ",using ip address instead.")
+					hostname = ip
```

Comment:
Would you mind combining this with a command line flag perhaps `-skip-reverse-dns` or something. We have missing reverse records that we do not want to add to prometheus. The feature is to only add those who have reverse records.

Reply:
Sure,
i have created a new pull request introducing a new flag/switch: https://github.com/FortnoxAB/prometheus-net-discovery/pull/16